apollo-portal/src/main/resources/static/user-manage.html

```diff
@@ -33,18 +33,82 @@
+                                </div>
+                                <button type="button" class="btn btn-default" ng-click="resetSearchUser()">{{'UserMange.Reset' | translate }}</button>
+                            </form>
+                            <div style="height: 15px"></div>
+                        </div>
+                        <table class="table table-bordered table-striped table-hover">
+                            <tr>
+                                <th style="text-align: left">{{'UserMange.UserName' | translate }}</th>
+                                <th style="text-align: left">{{'UserMange.UserDisplayName' | translate }}</th>
+                                <th style="text-align: left">{{'UserMange.Email' | translate }}</th>
+                                <th style="text-align: left">{{'UserMange.Enabled' | translate }}</th>
+                                <th style="text-align: left">{{'UserMange.Operation' | translate }}</th>
+                            </tr>
+                            <tr ng-repeat="user in filterUser" href="#" class="hover cursor-pointer">
+                                <td>{{ user.userId }}</td>
+                                <td>{{ user.name }}</td>
+                                <td>{{ user.email }}</td>
+                                <td>
+                                    <span class="label label-primary no-radius namespace-label" data-tooltip="tooltip"
+                                          data-placement="bottom"
+                                          ng-show="user.enabled === 1">{{('UserMange.Enable' | translate)}}
+                                    </span>
+                                    <span class="label label-warning no-radius cursor-pointer" data-tooltip="tooltip"
+                                          data-placement="bottom"
+                                          ng-show="user.enabled === 0">{{('UserMange.Disable' | translate)}}
+                                    </span>
+                                </td>
+                                <td>
+                                    <span class="btn btn-primary" ng-click="changeStatus('3', user)">
+                                        {{'UserMange.Edit' | translate }}
+                                    </span>
+                                    <span class="btn btn-primary" ng-click="changeUserEnabled(user)">
+                                        {{user.enabled === 1 ? ('UserMange.Disable' | translate) : ('UserMange.Enable' | translate) }}
+                                    </span>
+                                </td>
+                            </tr>
+                        </table>
+                    </div>
+                    <div ng-show="status==='2' || status==='3'">
+                        <form class="form-horizontal panel-body" name="appForm"
+                              valdr-type="App" ng-submit="createOrUpdateUser()">
```

Comment:
As we now have `add` and `edit` operations, is it better to distinguish them?
For example, if the admin adds a user that already exists, we need to prompt an error message like `User xxx already exists`?

Reply:
@zcy1010 What do you think of this? I understand it would involve more logic changes but it is worth it as this would greatly improve the user adding experience.